src/bin/cargo-miri.rs

```diff
@@ -369,13 +424,20 @@ path = "lib.rs"
         None => true,
         Some(target) => target == rustc_version::version_meta().unwrap().host,
     };
-    let sysroot = if is_host { dir.join("HOST") } else { PathBuf::from(dir) };
+    let sysroot = if is_host {
+        dir.join("HOST")
+    } else {
+        PathBuf::from(dir)
+    };
```

Comment:
Why'd it do this? The old line doesn't seem longer than 100 characters?

Reply:
I think it just hates single line `if` expressions

Reply:
It kept them elsewhere though.